vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/receivers/MultiFileBuffer.java

```diff
@@ -41,11 +41,18 @@
         implements MultiFileReceiver {
 
     private Map<String, FileData> files = new HashMap<>();
+	
+	public MultiFileBuffer(){}
```

Comment:
Missing a `super();` call here, otherwise the default file factory will not be initialized.

Also missing API docs, which could be copied from `AbstractFileBuffer`.